analyzer-core/src/main/java/ffb/analyzer/core/queue/Task.java

```diff
@@ -0,0 +1,43 @@
+package ffb.analyzer.core.queue;
+
+import java.util.function.Consumer;
+
+public class Task<T> {
```

Comment:
Do we need this to be generic? Can't `Task` just take `Object`? This would resolve all the 'raw usage of ...' errors in `PriorityQueueManager`.

Reply:
Discussed yesterday